src/foam/u2/Element.js

```diff
@@ -861,7 +885,13 @@ foam.CLASS({
       class: 'Object',
       name: 'id',
       transient: true,
-      factory: function() { return this.NEXT_ID(); }
+      // factory: function() { return this.NEXT_ID(); }
+      factory: function() {
```

Comment:
What is this for?

Reply:
You know what this is for :P sorry, forgot to remove it